### PR TITLE
make sure ids.log doesn't grow to large

### PR DESCRIPTION
**Pull request by devo1ution**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [x] I have linked this PR to any issues that it closes.

### CODE REVIEW

File: gatsby-node.ts [hunk 1]
```diff
@@ -24,6 +24,7 @@ try {
 
 // ideally problems would be its own query with
 // source nodes: https://www.gatsbyjs.com/docs/reference/config-files/gatsby-node/#sourceNodes
+if (fs.existsSync('ids.log')) fs.rmSync('ids.log');
```
Comment (SansPapyrus683) [inline gatsby-node.ts:27]: does this just delete ids.log?

Comment (devo1ution) [reply gatsby-node.ts:27]: yeah if it gets deleted if it already exists so basically it clears the log so it doesn't grow too big locally

**Comment by SansPapyrus683** [comment]
wait since when could you merge prs

**Comment by devo1ution** [comment]
😈

**Comment by SansPapyrus683** [comment]
no like seriously

**Comment by devo1ution** [comment]
idk i think its bc i got added to the webmaster team

**Pull request merged**